examples/l3fwd-graph/parse-args.c

```diff
@@ -174,14 +175,15 @@ static void
 print_usage(char *prog_name)
 {
     cne_printf("Usage: %s [-h] [-c json_file] <mode>\n"
-               "  <mode>         Mode types drop, tx-only, or [lb | loopback]\n"
-               "  -c <json-file> The JSON configuration file\n"
-               "  -C             Wait on unix domain socket for JSON or JSON-C file\n"
-               "  -d             More debug stats are displayed\n"
-               "  -D             JCFG debug decoding\n"
-               "  -V             JCFG information verbose\n"
-               "  -P             JCFG debug parsing\n"
-               "  -h             Display the help information\n",
+               "  <mode>          Mode types drop, tx-only, or [lb | loopback]\n"
+               "  -c <json-file>  The JSON configuration file\n"
+               "  -C              Wait on unix domain socket for JSON or JSON-C file\n"
+               "  -d              More debug stats are displayed\n"
+               "  -D              JCFG debug decoding\n"
+               "  -V              JCFG information verbose\n"
+               "  -P              JCFG debug parsing\n"
+               "  -L [info|debug] Enable debug or info Level Logging\n"
```

Comment:
```suggestion
               "  -L [level] Enable a logging level\n"
```

Reply:
Seems reasonable to change the help text.